.svnignore

```diff
@@ -70,6 +72,11 @@ extensions/**/*.png
 extensions/**/*.sass
 extensions/**/*.scss
 extensions/**/*.svg
+extensions/blocks/videopress/deprecated/v1
+extensions/shared/with-has-warning-is-interactive-class-names
+extensions/shared/components/upgrade-nudge
```

Comment:
If I recall correctly, the only thing we want to ship from `/extensions` is PHP. Is that right?

Any reason to remove these directories specifically, or do the rules above catch them?

Reply:
The rules above get rid of the files inside the folders, but we end up with empty folders pushed to SVN, which is not ideal. My goal here was to get rid of the folders entirely.

Reply:
This is a fine, I don't think it's worth holding up a change in it.

That said, this seems like it would be better handled where the svnignore is executed:

https://github.com/Automattic/jetpack/blob/a5d2f66b82beb819944a4d05bf58b0655bd69868/tools/build-jetpack.sh#L77
https://github.com/Automattic/jetpack/blob/ff16087b2825dc46bbe30b311121360fb06eb4b2/tools/deploy-to-master-stable-branch.sh#L76
https://github.com/Automattic/jetpack/blob/ee0fcec521f94cb2596ea4a8879350ca098457ae/tools/deploy-to-svn.sh#L85

The svnignore implementation is a good candidate for some reuse and iteration.

Are there _any_ empty plugin directories that need to be kept around?

Reply:
> That said, this seems like it would be better handled where the svnignore is executed:

That's a good idea. I just pushed c40eefcc95e413851eda51660f9313edc641c2fa to try this approach.